apps/rush-lib/src/logic/installManager/RushInstallManager.ts

```diff
@@ -32,21 +37,6 @@ import { AlreadyReportedError } from '../../utilities/AlreadyReportedError';
 import { LinkManagerFactory } from '../LinkManagerFactory';
 import { BaseLinkManager } from '../base/BaseLinkManager';
 
-/**
- * The "noMtime" flag is new in tar@4.4.1 and not available yet for \@types/tar.
- * As a temporary workaround, augment the type.
- */
-declare module 'tar' {
-  // eslint-disable-next-line @typescript-eslint/naming-convention
-  export interface CreateOptions {
-    /**
-     * "Set to true to omit writing mtime values for entries. Note that this prevents using other
-     * mtime-based features like tar.update or the keepNewer option with the resulting tar archive."
-     */
-    noMtime?: boolean;
-  }
-}
-
```

Comment:
As I explained with `IYarnLockfileParseResult`, just use a regular `import` with a TODO comment saying to convert to  "import type" later.  *Don't remove our type annotations.*